routes/index.js

```diff
@@ -0,0 +1,22 @@
+// 1. IMPORTACIONES
+
+const express    = require("express");
+const res        = require("express/lib/response");
+const router     = express.Router();
+
+//2. RUTEO
+router.get("/index", (req,res) =>{
+
+    res.render("index")
+} )
+
+router.get("/about", (req,res) => {
+    res.render("about")
+})
+
+router.get("/works", (req, rest) => {
```

Comment:
El error que me compartiste es por que en esta ruta le asignaste rest al parametro que se encarga de gestionar la respuesta del server solo hace falta quitar la t 👀 
```suggestion
router.get("/works", (req, res) => {
```